tests/integration_tests/test_trainer.py

```diff
@@ -239,3 +241,33 @@ def create_dataset(model: LudwigModel, size: int) -> RayDataset:
     assert train_ds.ds.num_blocks() == 2
     assert val_ds.ds.num_blocks() == 2
     assert test_ds.ds.num_blocks() == 2
+
+
+@pytest.mark.skipif(torch.cuda.device_count() == 0, reason="test requires at least 1 gpu")
+@pytest.mark.skipif(not torch.cuda.is_available(), reason="test requires gpu support")
```

Comment:
nit: It could be worth adding this test condition to `conftest.py` if we think we will re-use this, i.e.

```python
def gpu_available():
    return torch.cuda.device_count() == 0 and torch.cuda.is_available()

@pytest.mark.skipif(not gpu_available(), reason="test requires gpus")
```

Reply:
Agree with this, I think it would be right to address this as part of the re-enablement of GPU tests.